core/app/models/spree/stock/package.rb

```diff
@@ -62,7 +65,8 @@ def currency
       end
 
       def shipping_categories
-        contents.map { |item| item.variant.shipping_category }.compact.uniq
+        Spree::ShippingCategory.joins(products: :variants_including_master).
```

Comment:
@priyank-gupta - We should move includes(:shipping_methods) to where it is used.